config/default.go

```diff
@@ -43,17 +43,18 @@ var defaultConfig = &Config{
 	Registry: Registry{
 		Backend: "consul",
 		Consul: Consul{
-			Addr:          "localhost:8500",
-			Scheme:        "http",
-			KVPath:        "/fabio/config",
-			TagPrefix:     "urlprefix-",
-			Register:      true,
-			ServiceAddr:   ":9998",
-			ServiceName:   "fabio",
-			ServiceStatus: []string{"passing"},
-			CheckInterval: time.Second,
-			CheckTimeout:  3 * time.Second,
-			CheckScheme:   "http",
+			Addr:               "localhost:8500",
+			Scheme:             "http",
+			KVPath:             "/fabio/config",
+			TagPrefix:          "urlprefix-",
+			Register:           true,
+			ServiceAddr:        ":9998",
+			ServiceName:        "fabio",
+			ServiceStatus:      []string{"passing"},
+			CheckInterval:      time.Second,
+			CheckTimeout:       3 * time.Second,
+			CheckScheme:        "http",
+			CheckTLSSkipVerify: false,
```

Comment:
The field itself is defined in `config.Config` and not here. The value that you are setting in `defaultConfig` is automatically set by the compiler and therefore redundant. `false` is the default value for any `bool` variable.